frontend-new/src/consent/components/consentPage/consent.test.tsx

```diff
@@ -125,15 +135,21 @@ describe("Testing Data Protection Policy component", () => {
         // WHEN the component is rendered
         render(
           <HashRouter>
-            <DataProtectionAgreement />
+            <Consent />
           </HashRouter>
         );
 
         // WHEN the user accepts the terms and conditions
-        const checkBoxWrapper = screen.getByTestId(DATA_TEST_ID.ACCEPT_DPA_CHECKBOX);
-        expect(checkBoxWrapper).toBeInTheDocument();
-        const checkBoxInput = screen.getByRole("checkbox") as HTMLInputElement;
-        fireEvent.click(checkBoxInput);
+        const dpaCheckBoxWrapper = screen.getByTestId(DATA_TEST_ID.ACCEPT_DPA_CHECKBOX_CONTAINER);
+        expect(dpaCheckBoxWrapper).toBeInTheDocument();
+        const dpaCheckbox = dpaCheckBoxWrapper.getElementsByTagName("input")[0] as HTMLInputElement;
+        fireEvent.click(dpaCheckbox);
+
+        // AND Terms and Conditions are accepted
```

Comment:
it is different because right now we have two agreements.

1. Data Protection Agreement.
2. Terms and Conditions of using the app.

I have changed the comments to be more specific.

Thanks for spotting that